### PR TITLE
languages: add FULL translations (read full commit message)

### PR DESCRIPTION
**Pull request by Jolty95**
English: Robz8 and me (me for only a few strings)
Spanish: Jolty95
German: opa334
Portuguese: btuger
Italian: nop90
Traditional chinese: kane159 (孫小毛)
French: @mariogamer2 and @maorninja

Others still need a translation.

@Robz8 I need you to help me finding a new font (seriously, font is too burry some times) and something compatible with Chinese letters! :)
(PS: Maybe there are bugs, maybe not...who knows)

### CODE REVIEW

File: gui/source/languages/italian.h [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+#include <3ds/types.h>
+
+// ITALIAN
+static const char *const lang_IT[STR_MAX] = {
+	
+	/** GUI **/
+    "Ritorna al menu HOME",                                       // "Return to HOME Menu"
+    "AVVIA",                                                   // "START" (used on cartridge indicator)
+	"Nessuna cartuccia",                                       // "No cartridge"
+	"Unknown Cartridge", 										// "Unknown Cartridge"
```
Comment (Valeri0p) [inline gui/source/languages/italian.h]: Untranslated, it should be:
`"Cartuccia non riconosciuta", 										// "Unknown Cartridge"`

Comment (Jolty95) [inline gui/source/languages/italian.h]: Thanks for noticing that

Comment (Valeri0p) [inline gui/source/languages/italian.h]: Thank you for your work on this project ;)
There are some untranslated and misspelled strings, can I comment they?

Comment (Jolty95) [inline gui/source/languages/italian.h]: Yes, of course :)

File: gui/source/languages/italian.h [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+   
+    /** Settings: GUI **/
+    "Lingua",                                                   // "Language"
+    "Tema",                                                       // "Theme"
+    "Colore",                                                   // "Color"
+    "Colore del menu",                                         // "Menu Color"
+    "Visualizza il nome dei file",                                 // "Show filename"
+    "Contatore giochi",                                           // "Game counter"
+    "Sfonfo inferiore personalizzato",                       // "Custom bottom image"
+    "Aaggiorna automaticamente bootstrap",                               // "Auto-update bootstrap"
```
Comment (Valeri0p) [inline gui/source/languages/italian.h]: Typing error, should be:
`"Aggiorna automaticamente bootstrap",                               // "Auto-update bootstrap"`

File: gui/source/languages/italian.h [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    "Aggiorna automaticamente TWLoader",                           // "Auto-update to latest TWLoader"
+	
+    /** Settings: NTR/TWL_mode **/
+    "Seleziona la Flashcard",                                   // "Flashcard(s) select"
+    "LED arcobaleno",                                           // "Rainbow LED"
+    "Velocità CPU ARM9",                                       // "ARM9 CPU Speed"
+    "VRAM boost",                                               // "VRAM boost"
+    "Schermata di avvioi DS/DSi",                                       // "DS/DSi Boot Screen"
```
Comment (Valeri0p) [inline gui/source/languages/italian.h]: Another typing error, should be: 
`"Schermata di avvio DS/DSi",                                       // "DS/DSi Boot Screen"`

**Comment by maorninja** [comment]
lol, I forgot the comma. My bad. :P

**Comment by Jolty95** [comment]
Don't mind @maorninja , I forget a really big issue on GUI code hahaha :P I'm fixing asap

**Comment by maorninja** [comment]
BTW, can I have a build of the latest commit, just to test stuff?

**Comment by Jolty95** [comment]
Yeah, I'll send you by PM ;)

File: gui/source/languages/italian.h [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    "all'avvio.",                                                         // (empty)
+    "Scarica automaticamente l'ultima",                           // "Auto-download the CIA of the latest"
+    "versione del CIA TWLoader  all'avvio.",                               // "TWLoader version at launch."
+	
+    /** Settings: NTR/TWL_mode descriptions **/
+    "Seleziona una flashcard da usare",                           // "Pick a flashcard to use to"
+    "per avviare le ROM.",                                       // "run ROMs from it."
+    "Mosta luci arcobaleno sul",                               // "See rainbow colors glowing in"
+    "LED di notifica.",                                           // "the Notification LED."
+    "Impostastione affichè TWL elimini",                       // "Set to TWL to get rid of lags in some games."
```
Comment (Valeri0p) [inline gui/source/languages/italian.h]: I think the older translation is better, this is indecipherable Dx
`"Imposta su TWL per eliminare",                       // "Set to TWL to get rid of lags in some games."` continue in next comment...

File: gui/source/languages/italian.h [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+    "il lag su alcuni giochi.",                                 // (empty)
```
Comment (Valeri0p) [inline gui/source/languages/italian.h]: `"il lag in alcuni giochi.",                                 // (empty)`

File: gui/source/languages/italian.h [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+    "Predefinito",                                               // "Default"
+	
+    /** Sub-theme **/
+    "Seleziona sotto-tema: DSi Menu",                           // "Sub-theme select: DSi Menu"
+    "Seleziona sotto-tema: R4",                                   // "Sub-theme select: R4"
+    "Seleziona sotto-tema: Wood",                               // "Sub-theme select: Wood"
+    "Non esistono sotto-temi per questo tema.",                   // "No sub-themes exist for this theme."
+	
+    /** Settings others minor strings **/
+    "A/B: Salva a torna a precedente",                           // "A/B: Save and Return"
```
Comment (Valeri0p) [inline gui/source/languages/italian.h]: Typing error
`"A/B: Salva e torna a precedente",                           // "A/B: Save and Return"`

**Comment by Valeri0p** [comment]
Ok, I noticed the update for Italian, but still there are some little errors...

**Comment by Jolty95** [comment]
@Robz8 I was merging @Valeri0p 's review! 
Also fixing a bug. I'll create another PR using main.cpp and italian.h if you wish :)

**Comment by maorninja** [comment]
Okay, I know a guy that knows Arabic. @FruitSmasher is his name. Sadly, IDK if he has a gbatemp account, so you might need to talk to him by email.

**Pull request merged**